modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/PartitionListener.java

```diff
@@ -544,23 +550,24 @@ public void onConfigurationCommitted(
                 storage.committedGroupConfiguration(config);
                 storage.lastApplied(lastAppliedIndex, lastAppliedTerm);
 
-                if (!enabledColocation()) {
-                    updateTrackerIgnoringTrackerClosedException(storageIndexTracker, config.index());
```

Comment:
Why is it moved outside?

Reply:
1. Why was it inside in the first place?
2. There's another `!enabledColocation` check below, so I just unified them into one.